meteor/server/main.js

```diff
@@ -0,0 +1,350 @@
+import { Meteor } from 'meteor/meteor';
+import uuid from "uuid";
+import '/imports/settings.js';
+import '/imports/collections.js'
+import './methods.js';
+import './mozdef.js';
+
+Meteor.startup(() => {
+  // code to run on server at startup
+
+
+if (Meteor.isServer) {
+    //Publishing setups
```

Comment:
Everything from line 13 to line 255 can be removed. 
These are published via collections.js in the imports directory. 
I've tested the removal in my own environment, and then tested that all the functionality still exists.
This will get rid of the "ignoring publish.. " error messages when starting meteor.

Reply:
good catch! missed the 'cut' part of copy/paste ;-]